libavformat/segment.c

```diff
@@ -956,6 +968,10 @@ static int seg_write_packet(AVFormatContext *s, AVPacket *pkt)
         seg->cur_entry.index = seg->segment_idx + seg->segment_idx_wrap * seg->segment_idx_wrap_nb;
         seg->cur_entry.start_time = (double)pkt->pts * av_q2d(st->time_base);
         seg->cur_entry.start_pts = av_rescale_q(pkt->pts, st->time_base, AV_TIME_BASE_Q);
+
+        av_log(s, AV_LOG_DEBUG, "pkt->pts=%"PRId64", start_time=%f, seg->reset_timestamps=%d, st->start_time=%"PRId64", rescale=%"PRId64,
```

Comment:
I would make this the main log and include the items in the previous av_log() - since there are no conditional paths between them.  Better to have one log line instead of two.